### DISCUSSION TITLE
Are the docker image versioned?

### DISCUSSION THREAD

**Discussion by Oren-i** [onekey-sec/unblob#438] [Q&A] [answered]
Is latest the only tag for http://ghcr.io/onekey-sec/unblob or are there other tags?

**Comment by kissgyorgy** [comment]
Hello!
We don't *yet* have a proper release schedule, we are just keeping master stable and using the latest version in production, so the Docker images are not yet tagged with versions.

Later we will have a proper release schedule with point versions and proper tags for the Docker images, but I can't tell you an ETA.
Using only the latest version should not be a problem, however.

**Comment by qkaiser** [comment, marked as answer]
docker images are now versioned, the latest image as of today is `24.2.20`.

You can pull it with:

```
docker pull ghcr.io/onekey-sec/unblob:24.2.20
```
